roles/rocketchat-nginx/handlers/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+- name: restart nginx
+  become: yes
+  service: name=nginx state=restarted enabled=yes
```

Comment:
Bitte keinen MIX von JSON und YAML also bitte:
```
service:
  name: nginx
  state: restarted
  enabled: true
```

Reply:
Wontfix. Das ist kein Json.

Reply:
Geschenkt ist kein JSON, aber ist nicht so gut lesbar, wie meine Zeilen. Nicht ohne Grund macht man auch in Programmiersprachen immer pro Befehl eine Zeile.